README.md

```diff
@@ -93,26 +130,35 @@ source .venv/bin/activate
 pip install -r requirements.txt
 ```
 
-In CML or CDSW, no virtual env is necessary. Instead, inside a JupyterLab Session (with at least 2 vCPU / 16 GiB Memory), simply run through the Setup cells in the Jupyter Notebook. 
+-----------
 
-### Running the evaluation script
 
-The [scripts/evaluate.py](scripts/evaluate.py) script requires the [scripts/config.txt](scripts/config.txt) file which handles all the input arguments. First specify your choices in the config file, and then run the following in an open Session: `!python3 scripts/evaluate.py @scripts/config.txt`
+## Execution 
 
-Alternatively, this script can also be run automatically with the **Jobs** abstraction by first clicking on **Jobs**, then **New Job**, and then selecting `scripts/evaluate.py` under **Script**. Enter `@scripts/config.txt` under **Arguments** and ensure that the job is given at least 2vCPU/16GiB of resources. Then click **Create Job**. You can now run the job as often as you like or schedule the job at your convenience. 
+Launch a JupyterLab Session with at least 16 GiB of memory and 2 vCPUs, on CML, or with similar resources on your own system.
 
------------
-### Notes on resources: 
+### Jupyter notebook
 
-Video data are rich and complex and, as such, require a considerable amount of storage and memory considerations. 
+The Jupyter notebook is contains instructions for running it.
 
-#### Storage
-The Kinetics video data used in this repo is only a fraction of what is available for model evaluation. Downloading the full, raw dataset would require 30-60GiB (depending on whether you download the `validation` or `test` set). Our `KineticsLoader` class converts these raw videos to NumPy arrays and caches them for re-use. These cached, pre-processed video clips require even more storage -- up to a TB for the full validation set. We recommend starting with only a few hundred videos and assessing your storage capabilities before running a model evaluation on the full Kinetics `test` or `validation` sets. 
 
-#### Memory
-In addition to the storage requirements, processing these data through a pre-trained CNN model also requires considerable RAM. We recommend a minimum of 16GiB which is enough to process a batch of 8 videos. Large batch sizes will require additional resources. 
+### Evaluation script
+
+The [scripts/evaluate.py](scripts/evaluate.py) script requires the [scripts/config.txt](scripts/config.txt) file which handles all the input arguments. You can use the default values that come with the repository, or specify your own values, in the config file. Execution is then done by executing this command on a terminal of the open session, at the root of the repository:
+
+```console
+$ python3 scripts/evaluate.py @scripts/config.txt
```

Comment:
While this is one way to run this script, it is not the ideal way that we recommend users to interact with scripts of this nature. We always recommend they use a standard open Session and run 

```python
!python3 scripts/evaluate.py @scripts/config.txt
```

As I had it before.

Reply:
In your original instructions, there was this line:
"then run the following in an open Session: `!python3 scripts/evaluate.py @scripts/config.txt`"

By "an open Session" do you mean "inside a Jupyter notebook"? 

The reason I ask is that a Session has both terminals as well as Jupyter notebooks available. Since Python scripts are more typically executed on terminals, I thought that's what you had meant (and I that the exclamation mark was a typo).

If your instruction was to execute the command inside a notebook, then, why is that the recommended way, instead of execution on a terminal?